### PR TITLE
newdevice: Basic hanwha techwin support

### PR DESCRIPTION
**Pull request by aldemira**
Also known as Samsung iPolis Cameras

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7397)

### CODE REVIEW

**Comment by laf** [comment]
Using the vendor name usually isn't a good OS name. Is iPOLiS not better?

**Comment by aldemira** [comment]
Of course, I didn't know what to write there since it's basically a branded linux.

File: includes/definitions/ipolis.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+os: ipolis
+text: 'Hanwha Techwin'
+type: camera
+icon: hanwhatechwin
+over: 
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.36849
```
Comment (laf) [inline includes/definitions/ipolis.yaml]: One last thing from me. That's quite a low level OID to use. I can see the full one is `.1.3.6.1.4.1.36849.1.2.88` - is it worth doing `.1.3.6.1.4.1.36849.1.2.` or `1.3.6.1.4.1.36849.1.`?

File: tests/snmpsim/ipolis.snmprec [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|SAMSUNG iPolis H.264 Premium Level IP Camera
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.36849
```
Comment (laf) [inline tests/snmpsim/ipolis.snmprec]: This needs to be the full oid: `.1.3.6.1.4.1.36849.1.2.88`

Comment (aldemira) [reply tests/snmpsim/ipolis.snmprec]: Mines were .1.3.6.1.4.1.36849.1.2.91 so I wrote that

**Pull request merged**